Make fly.io public (#18537)
DataDog/integrations-core: fly_io/README.md

# Agent Check: Fly.io

<div class="alert alert-warning">

> This integration is in public beta and should be enabled on production workloads with caution.

Reviewer: ```suggestion
This integration is in public beta. Use caution if enabling it on production workloads.
```
